lib/elixir_console_web/live/console_live/command_input_component.ex

```diff
@@ -78,23 +100,23 @@ defmodule ElixirConsoleWeb.ConsoleLive.CommandInputComponent do
     {:noreply, assign(socket, reset_input: true)}
   end
 
-  defp get_previous_history_entry([], _counter), do: {[], 0}
+  defp get_previous_history_entry([], _counter), do: {"", 0}
```

Comment:
I don't know why these functions returned lists with zero or one element but it didn't look necessary

Reply:
Not sure, but change makes sense :+1: